### PR TITLE
Improves formatting for mobile and forced vertical orientation

### PR DESCRIPTION
**Pull request by thomasbanghart**
Addresses issues with vertical formatting experienced by some users. 

* Adds `Orientation` option which allows users to select `Auto`, `Horizontal`, or `Vertical` layouts.
**We recommend using `Vertical` or `Auto` orientations for dashboards that will be primarily viewed on mobile devices**.

Screen Shot 2020-07-15 at 9 53 11 AM

* Addresses `flex-box` shrinking on Safari/iOS devices that "squeezed" containers together when in vertical layout. 

* Makes font-size mobile friendly

### CODE REVIEW

**Comment by thomasbanghart** [comment]
So rather than adding some margin or padding, justify-content makes sure that the top values aren't hidden under the parent container when the tile is too small. This seems to do it but please let me know if it looks off. Also adds an option to choose orientation

File: src/multiple_value/multiple_value.js [hunk 1]
```diff
@@ -25,13 +24,15 @@ const DataPointGroup = styled.div`
   text-align: center;
   width: 100%;
   display: flex;
+  flex-shrink: ${props => props.layout === 'horizontal' ? 'auto' : 0 };
```
Comment (noahamac) [inline src/multiple_value/multiple_value.js:27]: Really nice find here with this property & supportability testing

File: src/multiple_value/multiple_value.js [hunk 2]
```diff
@@ -155,7 +156,7 @@ class MultipleValue extends React.PureComponent {
   }
 
   getWindowSize = () => {
-    return window.innerWidth > window.innerHeight ? window.innerWidth : window.innerHeight;
+    return Math.max(window.innerWidth, window.innerHeight);
```
Comment (noahamac) [inline src/multiple_value/multiple_value.js:159]: Leavin' the place better than you found it! :D

File: src/multiple_value/multiple_value.js [hunk 3]
```diff
@@ -214,17 +217,23 @@ class MultipleValue extends React.PureComponent {
             }
             return (
 
-              <DataPointGroup comparisonPlacement={compDataPoint && config[`comparison_label_placement_${compDataPoint.name}`]} 
-              key={`group_${dataPoint.name}`}>
+              <DataPointGroup
```
Comment (noahamac) [inline src/multiple_value/multiple_value.js:220]: Thanks for cleaning up

File: src/multiple_value/multiple_value.js [hunk 4]
```diff
@@ -195,12 +197,13 @@ class MultipleValue extends React.PureComponent {
...
     let CONFIG = this.props.config;
+    
 
     return (
       <DataPointsWrapper
-        layout={this.state.groupingLayout}
+        layout={config['orientation'] === 'auto' ? this.state.groupingLayout : config['orientation']}
         font={config['grouping_font']}
-        style={{fontSize: `${this.state.fontSize}px`}}
+        style={{fontSize: `${this.state.fontSize}em`}}
```
Comment (noahamac) [inline src/multiple_value/multiple_value.js:206]: So this is the one thing I wanted to check on -- 'vh' or 'vw' is not suitable/preferable here? Will defer to your judgement here :+1:

File: src/multiple_value/multiple_value_container.js [hunk 5]
```diff
@@ -12,6 +12,21 @@ const baseOptions = {
     section: 'Style',
     default: "",
     order: 0,
+    display_size: 'half'
+  },
+  orientation: {
```
Comment (noahamac) [inline src/multiple_value/multiple_value_container.js:17]: :heart_eyes_cat:

**Pull request merged**